packages/discord.js/src/client/Client.js

```diff
@@ -166,11 +189,33 @@ class Client extends BaseClient {
      */
     this.application = null;
 
+    /**
+     * The latencies of the WebSocketShard connections
+     * @type {Collection<number,number>}
+     */
+    this.pings = new Collection();
+
+    /**
+     * The last time a ping was sent (a timestamp)
+     * @type {?number}
+     */
+    this.lastPingTimestamp = null;
+
     /**
      * Timestamp of the time the client was last {@link Status.Ready} at
      * @type {?number}
      */
     this.readyTimestamp = null;
+
+    /**
+     * An array of queued events before this Client became ready
+     * @type {Object[]}
+     * @private
+     * @name Client#packetQueue
+     */
+    Object.defineProperty(this, 'packetQueue', { value: [] });
```

Comment:
I'd opt for a different name here, packet is quite ambiguous, at a glance it feels like it could mean "send" as well.

Maybe `incomingEventQueue`